src/controller.js

```diff
@@ -36,6 +36,21 @@ export default class Controller {
     Rails.fire(_element(query), 'submit')
   }
 
+  url() {
+    return window.location.href
+  }
+
+  urlParams() {
```

Comment:
is this method really useful as public API?

Reply:
Yes it probably would be better to change it to private

Reply:
+1